lib/pf/Authentication/Source/SAMLSource.pm

```diff
@@ -13,6 +13,8 @@ Model for a SAML source
 use pf::Authentication::constants;
 use pf::constants::authentication::messages;
 use pf::constants;
+use pf::constants::saml;
```

Comment:
Are these 2 lines really necessary?

Reply:
I added those to stop the warnings on a compile test.
But as long as we guarantee that the libraries pf::constants::saml and pf::config are loaded before we use those constants and the config then no we do not need those two lines.

Reply:
Lets leave them in then